.rubocop_todo.yml

```diff
@@ -20,3 +37,10 @@ InternalAffairs/NodeMatcherDirective:
     - 'lib/rubocop/cop/betterment/spec_helper_required_outside_spec_dir.rb'
     - 'lib/rubocop/cop/betterment/timeout.rb'
     - 'lib/rubocop/cop/betterment/unscoped_find.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct-all.
+Rails/NegateInclude:
```

Comment:
The `Array#exclude?` method is not available when running tests, since we don't load Rails when running tests.